datastructure/static_range_sum/info.toml

```
@@ -0,0 +1,22 @@
+title = 'Static Range Sum'
+timelimit = 2.0
```

```suggestion
timelimit = 5.0
```
特に理由はないんですが、TLは5sか10sという風潮です